tools-local/Microsoft.ML.AutoML.SourceGenerator/Microsoft.ML.AutoML.SourceGenerator.csproj

```diff
@@ -12,13 +12,17 @@
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.2" PrivateAssets="all" />
     <PackageReference Include="System.CodeDom" Version="5.00" />
     <PackageReference Include="System.Text.Json" Version="$(SystemTextJsonVersion)" PrivateAssets="All" GeneratePathProperty="true" />
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version ="6.0.0" PrivateAssets="All" GeneratePathProperty="true" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" PrivateAssets="All" GeneratePathProperty="true" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="6.0.0" PrivateAssets="All" GeneratePathProperty="true" />
+    <PackageReference Include="System.Text.Encodings.Web" Version="6.0.0" PrivateAssets="All" GeneratePathProperty="true" />
```

Comment:
If you do private assets here they won't show up as nuget dependencies. Is that ok?

Reply:
Sounds good to me.

Reply:
### Resolved